frontend/src/components/PrevNext.js

```diff
@@ -1,16 +1,52 @@
 import React from 'react';
 import Data from './data.json'
-
+import './styles/BodyStyle.css'
+
+function openCloseAnswer(e)
+{
+  let element = e.target;
+  console.log(element)
+  if(element.tagName.toLowerCase() !== "div")
+    element = element.parentElement
+  element = element.nextElementSibling
+  
+  
+  const id = element.getAttribute("id").replace("A", "")
+  if(element.getAttribute("visible").toLowerCase() === "true")
+  {  
+        element.setAttribute("visible", "false");
+        element.style.display = "none";
+        document.getElementById("Qs"+id).innerHTML = "^"
+  }
+  else
+  {  
+        element.setAttribute("visible", "true");
+        element.style.display = "block";
+        document.getElementById("Qs"+id).innerHTML = "x"
+  }
+}
 
 function PrevNext() {
+  let questionsAnswers = []
+  const size = Math.min(Data.Questions.length, Data.Answers.length) 
+  for(var s_itr = 0; s_itr < size; s_itr++)
+    questionsAnswers.push({"question":Data.Questions[s_itr].text, "answer":Data.Answers[s_itr].text, "id":s_itr+1})
+
+  let  questionsAnswersHTML=questionsAnswers.map((questionAnswer)=> <div key = { Math.random()} className = "question-answer" >
```

Comment:
i would have prefer that the key it's actualy the index of the `questionAnswer`.. so am sugesting : 
```javascript
let  questionsAnswersHTML = questionsAnswers.map( (questionAnswer, index) => <div key = { index } className = "question-answer" >
```
instead... but you are free to still use `Math`